gflownet.py

```diff
@@ -716,6 +780,7 @@ def flowmatch_loss(self, it, batch):
             with torch.no_grad():
                 next_q = self.target(sp)
         else:
+            # TODO: potentially mask invalid actions next_q
```

Comment:
I suggest that we directly add the the mask in the training batch information, so that we don't have to compute it another time ! And then it is straightforward to mask the invalid actions in the loss function (if the mask is 1 for valid action and 0 for invalid after_masking_actions = torch.where(masks == 1, next_q, -loginf_tensor))

Reply:
Yes, indeed.